From a688f0c8cb2ab4fa8502e2ddae0c41250df4209f Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:32:46 -0400
Subject: [alsa] hda/tegra: Fix MODULE_DEVICE_TABLE typo

Message-id: <140972823573-302-patchset-jkysela@redhat.com>
Patchwork-id: 90998
O-Subject: [RHEL 7.1 PATCH 301/442] ALSA: hda/tegra - Fix MODULE_DEVICE_TABLE typo.
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

I missed a rename during the review process.  Fix the
MODULE_DEVICE_TABLE to match the structure.

Signed-off-by: Dylan Reid <dgreid@chromium.org>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: f73387cb6b1f1534e56669e6aecefae1be867f59)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/hda_tegra.c b/sound/pci/hda/hda_tegra.c
index e472bc8..086fd06 100644
--- a/sound/pci/hda/hda_tegra.c
+++ b/sound/pci/hda/hda_tegra.c
@@ -496,7 +496,7 @@ static const struct of_device_id hda_tegra_match[] = {
  { .compatible = "nvidia,tegra30-hda" },
  {},
 };
-MODULE_DEVICE_TABLE(of, tegra_platform_hda_match);
+MODULE_DEVICE_TABLE(of, hda_tegra_match);
 
 static int hda_tegra_probe(struct platform_device *pdev)
 {
-- 
1.7.1